src/src/PluginStructs/P128_data_struct.h

```diff
@@ -2329,13 +2329,15 @@ struct P128_data_struct : public PluginTaskData_base {
 
   unsigned long counter20ms = 0,
                 starttime[ARRAYSIZE],
-                starttimerb,
-                maxtime = 0;
+                starttimerb = 0,
+                maxtime     = 0;
 
   P128_modetype mode     = P128_modetype::Off,
                 savemode = P128_modetype::Off,
```

Comment:
What is being saved in this save mode?
Energy?

Reply:
You can turn the effect/action `on` and `off`, when turning it back on, the saved mode is activated.